### PR TITLE
Add methods to get/set log file description and test properties.

### PR DESCRIPTION
**Pull request by peterilbergNI**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/flexlogger-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

This pull request
- adds methods to get and set the log file description
- adds a method to set several test properties at once
- adds example projects to get and set the log file description
- adds example projects to get and set test properties

### Why should this Pull Request be merged?

Customers have requested methods to get/set the log file description and to set several test properties at once.

### What testing has been done?

I have run added new tests for the new methods. I have run the automated tests and the new example projects.

- [x] I have run the automated tests (required if there are code changes)

### CODE REVIEW

**Comment by chadallee** [comment]
Bump the version to `"FlexLogger 2023 Q2+"` in [README.rst](https://github.com/ni/niflexlogger-automation-python/blob/main/README.rst)

**Review by DevinKoopmans — CHANGES_REQUESTED** [review]
Looks good aside from the one comment I posted

**Review by ccaltagi — APPROVED** [review]
This should be good to go after the changes requested by Chad and Devin.

File: src/flexlogger/automation/_project.py [hunk 1]
```diff
@@ -145,6 +145,8 @@ def save(self) -> None:
 
         Raises:
             FlexLoggerError: if saving the project fails due to a communication error.
+            If there is no communication error with the FlexLogger application, this function will not raise an
+            exception, but instead return a boolean indicating if the project was properly saved.
```
[the remark this answers is not in the document]
Comment (peterilbergNI) [reply src/flexlogger/automation/_project.py]: I don't understand how it came back either. I have removed it again and I've updated the version to 2023 Q2+ as requested by Chad.

**Review by chadallee — APPROVED** [review]

**Pull request merged**